### DISCUSSION TITLE
Adding more tests and increasing the code coverage

### DISCUSSION THREAD

**Discussion by Sonichigo** [nocodb/nocodb#7209] [Q&A] [answered]
I was going through the codebase, to see how ways to contribute to the nocodb. I noticed that there are tests written in playwright. use a tool called Keploy to for testing and generating datamocks, I think it would be a nice to have some more-testcases. Can i contirbute in this way?

**Comment by o1lab** [comment, marked as answer]
<img width="962" alt="image" src="https://github.com/nocodb/nocodb/assets/5435402/6b0c4061-e9ae-4848-9ef5-19ed9e9554b0">
